models/lido/liquidity/base/lido_liquidity_base_uniswap_v3_pools.sql

```diff
@@ -63,7 +63,7 @@ with
     {% if not is_incremental() %}
     WHERE DATE_TRUNC('day', p.minute) >= DATE '{{ project_start_date }}'
     {% else %}
-    WHERE DATE_TRUNC('day', p.minute) >= DATE_TRUNC('day', NOW() - INTERVAL '1' day)
+    WHERE {{ incremental_predicate('p.minute') }}
```

Comment:
similar to the changes made here to use incremental predicate on source reads, we also leverage them for filtering the target data. this allows the generated merge statement to filter both to same time period and speed up processing. i won't consider a blocker for now, but it'd be nice to see a follow up PR soon that implements on all lido models. looks like this:
https://github.com/duneanalytics/spellbook/blob/main/models/_sector/dex/trades/arbitrum/platforms/airswap_arbitrum_base_trades.sql#L9

be sure the column name passed in matches target spell column name, sometimes it changes from source to target